BUILD

```diff
@@ -18,64 +19,37 @@ touch $@
     stamp = True,
 )
 
-alias(
-    name = "android_pom",
-    actual = "//library/kotlin/src/io/envoyproxy/envoymobile:android_aar_pom",
-)
-
 alias(
     name = "android_aar",
-    actual = "//library/kotlin/src/io/envoyproxy/envoymobile:android_aar",
-)
-
-alias(
-    name = "android_javadocs",
-    actual = "//library:javadocs",
-)
-
-alias(
-    name = "android_sources",
-    actual = "//library:sources_jar",
+    actual = "@envoy_mobile//library/kotlin/src/io/envoyproxy/envoymobile:android_aar_only_aar",
+    visibility = ["//visibility:public"],
 )
 
 genrule(
-    name = "android_dist",
+    name = "android_zip",
     srcs = [
-        "android_aar",
+        "@envoy_mobile//library/kotlin/src/io/envoyproxy/envoymobile:android_aar",
     ],
-    outs = ["stub_android_dist_output"],
-    cmd = """
-cp $(location :android_aar) dist/envoy.aar
-chmod 755 dist/envoy.aar
-touch $@
-""",
+    outs = ["envoy_mobile.zip"],
+    cmd = "$(location @envoy_mobile//bazel:zipper) fc $@ $(SRCS)",
     stamp = True,
```

Comment:
Seems like you probably shouldn't need stamp here 🤔

Reply:
What does that do actually?